administrate/metrics/overview.md

```diff
@@ -182,22 +168,20 @@ In the following example, we get the `accessLogs` status codes and create a GTS
 
 An example using the provided Clever Cloud macro to straightforward access to the access logs input byte :
 
-```bash
-  '<READ TOKEN>' { 'app_id'  'id' } 'bIn' NOW 1 h  @clevercloud/fetch_accessLogs_key_v0
+```WarpScript
```

Comment:
Not sure if we should specify a text type that doesn't exist instead of nothing :thinking: